release/nightly_tests/dataset/gpu_batch_inference.py

```diff
@@ -1,30 +1,43 @@
 from typing import Dict
-import click
+import argparse
 import time
-import os
 
 import numpy as np
 import torch
 from torchvision.models import resnet50, ResNet50_Weights
 
-from benchmark import Benchmark
+from benchmark import Benchmark, BenchmarkMetric
 import ray
 from ray.data import ActorPoolStrategy
 
 
-@click.command(help="Run Batch prediction on Pytorch ResNet models.")
```

Comment:
Change `click` usage to `argparse` instead, because it was breaking the reporting through the `Benchmark` utility specifically for `benchmark.read_fn()`.

Reply:
after the `main` method is finished and returns, the rest of the benchmarking code in `benchmark.run_fn()` and the subsequent `benchmark.write_results()` would not run. I didn't have time to investigate deeply why the code early exits after `main` successfully returns, but I was able to confirm with the release test runs that this is the reason why metrics weren't successfully exported and printed.